assets/js/views/_paymentCategories.js

```diff
@@ -0,0 +1,63 @@
+OpenDisclosure.PaymentCategoryView = Backbone.View.extend({
+  initialize: function(options) {
+    this.options = options;
+    _.bindAll(this, 'displayDetails');
+    this.render();
+  },
+
+  render: function() {
+    var attributes = this.options.attributes;
+    this.$el.empty();
+    this.$el.append('<h2>Total Payments by Category</h2>');
+    // Create the data table.
+    var data = new google.visualization.DataTable();
+    this.data = data;
+    data.addColumn('string', 'Category');
+    data.addColumn('number', 'Amount');
+    data.addColumn('string', 'Code');
+    _.each(this.collection, function( c ){
+      data.addRow([c.attributes.text, c.attributes.amount, c.attributes.code]);
+    });
+
+    pieChart = new Backbone.GoogleChart({
+      chartType: 'PieChart',
+      options: {
+       // 'title':'Total Contributions by Category',
+       // 'titleTextStyle':{'fontSize':30, 'fontName':'Crete Round', 'color':'#555555'},
+       'backgroundColor': '#E9E9E9',
+       'chartArea': {
+         'width': 300
+       },
+       'width': 300,
+       'height': 250,
+       'sliceVisibilityThreshold': 0
+      },
+      dataTable: data,
+    });
+
+    // This is pretty hacky, but it's a way to center the chart without reaching
+    // into its internals.
+    pieChart.render();
```

Comment:
You should ask the author of this code, thats where I copied it from:
19e9730b assets/js/views/_categoryView.js (Tom Dooner    2014-09-01 18:16:18 -0700 36)     // This is pretty hacky, but it's a way to center the chart without reaching
19e9730b assets/js/views/_categoryView.js (Tom Dooner    2014-09-01 18:16:18 -0700 37)     // into its internals.
19e9730b assets/js/views/_categoryView.js (Tom Dooner    2014-09-01 18:16:18 -0700 38)     pieChart.render();